retrace-server.spec.in

```diff
@@ -183,6 +188,7 @@ fi
 %{_bindir}/coredump2packages
 %{python3_sitelib}/retrace/
 %{_datadir}/%{name}/
+%{_localstatedir}/lib/retrace/
```

Comment:
`%dir %attr(0755,retrace,retrace) %{_sharedstatedir}/%{name}`

Otherwise the dir is owned by root:
`drwxr-xr-x. 2 root root 4096 Dec 12 17:46 /var/lib/retrace`